### PR TITLE
[ADD] issue and pull request templates.

### LINKED ISSUE #293: Add issue templates
Like https://github.com/neurobagel/digest/issues/new/choose

### PR DESCRIPTION
**Pull request by mathdugre**
fix #293

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+-->
+
+<!-- 
+Please indicate after the # which issue you're closing with this PR, if applicable.
+If the PR closes multiple issues, include "fix" before each one is listed.
+You can also link to other issues if necessary, e.g. "See also #1234".
+
+https://help.github.com/articles/closing-issues-using-keywords
+-->
+- fix #
```
Comment (michellewang) [inline .github/pull_request_template.md:16]:
```suggestion
- Fix #
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    validations:
+      required: false
+
+  # - type: textarea
+  #   attributes:
+  #     label: Version
+  #     description: |
+  #       Which version did you encounter this bug on?
+  #     value: |
+  #       - version:
```
Comment (michellewang) [inline .github/ISSUE_TEMPLATE/bug_report.yml:62]: Uncomment this? For when we start making releases (hopefully soon!)

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+-
+-
+
+<!-- To be checked off by reviewers -->
+## Checklist
+_This section is for the PR reviewer_
+
+- [ ] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`)
+- [ ] PR links to GitHub issue with mention `fix #XXXX`
```
Comment (michellewang) [inline .github/pull_request_template.md:32]:
```suggestion
- [ ] PR links to GitHub issue with mention `Fix #XXXX`
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Changes proposed in this pull request:
+
+-
+-
+
+<!-- To be checked off by reviewers -->
+## Checklist
+_This section is for the PR reviewer_
+
+- [ ] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`)
```
Comment (michellewang) [inline .github/pull_request_template.md:31]: We should remove `[MODEL]` and probably `[INF]` too from the prefix list

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        If necessary give information about your setup.
+      value: |
+        - OS:
+        - Python/Node version:
```
Comment (michellewang) [inline .github/ISSUE_TEMPLATE/bug_report.yml:52]:
```suggestion
        - Python version:
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
+name: Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Fill in this template to report a bug
+
+title: "[BUG] "
+
+labels: ["type:bug"]
```
Comment (michellewang) [inline .github/ISSUE_TEMPLATE/bug_report.yml:10]: For now the repo only has the `bug` label (not `type:bug`)

```suggestion
labels: ["bug"]
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,43 @@
+name: Feature request
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Suggest an idea for this project
+
+title: "[ENH] "
+
+labels: ["feedback:feature request"]
```
Comment (michellewang) [inline .github/ISSUE_TEMPLATE/feature_request.yml:10]: `feedback:feature request` is currently not an existing label. For now we can remove this, though we might want a longer discussion about issue labels in the future

**Review by michellewang — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (mathdugre) [reply .github/ISSUE_TEMPLATE/bug_report.yml:10]: Modified to the new labels we created

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 7 above)
Comment (mathdugre) [reply .github/ISSUE_TEMPLATE/feature_request.yml:10]: Modified to the new labels we created

**Comment by mathdugre** [comment]
Closing here, and reopening in the org https://github.com/nipoppy/.github repo (see nipoppy/.github#1)

**Pull request closed without merging**